cypress/support/step_definitions/dashboard/list_settings/a_list_exists_with_users.js

```diff
@@ -4,28 +4,40 @@ Given("a list exists with users", () => {
   };
 
   cy.task("db", {
-    operation: "list.upsert",
+    operation: "list.findUnique",
```

Comment:
why did this need changing?

Reply:
Took me a while to remember but I think I've got it. Basically the code before was overwiritng whatever was already in jsonData with 

```
{
  users: ["smoke@cautionyourblast.com", "smoke+1@cautionyourblast.com", "smoke+2@cautionyourblast.com"],
}
 ```
 
 So the update was made to basically keep what was there before (annual review data etc...) and add the users data to it as well.